### PR TITLE
SPOI-7818 #resolve #comment Remove dimensions code duplication betwee…

### PR DESCRIPTION
**Pull request by brightchen**
…n megh and malhar

### CODE REVIEW

File: library/src/test/java/com/datatorrent/lib/io/fs/FileSplitterBaseTest.java [hunk 1]
```diff
@@ -97,7 +97,13 @@ protected void starting(org.junit.runner.Description description)
     @Override
     protected void finished(Description description)
     {
-      TestUtils.deleteTargetTestClassFolder(description);
+      try {
+        //got RuntimeException: java.io.IOException: Unable to delete directory target/com.datatorrent.lib.io.fs.FileSplitterBaseTest/testSplitterInApp
+        //in build server and sometime in local machine
+        TestUtils.deleteTargetTestClassFolder(description);
```
Comment (tweise) [inline library/src/test/java/com/datatorrent/lib/io/fs/FileSplitterBaseTest.java]: How about a deleteQuietly function if this is frequently needed?

**Comment by tweise** [comment]
Please create a JIRA that describes what this change is about (topN support?) and change the PR and commit message.

File: library/src/test/java/com/datatorrent/lib/io/fs/FileSplitterBaseTest.java (hunk 1 above)
Comment (brightchen) [inline library/src/test/java/com/datatorrent/lib/io/fs/FileSplitterBaseTest.java]: deleteQuietly is method of FileUtils, which means we need to duplicate the logic of "new File("target/" + description.getClassName())"

Comment (brightchen) [inline library/src/test/java/com/datatorrent/lib/io/fs/FileSplitterBaseTest.java]: This is in fact is not related to this task.
Maybe it related to "APEXMALHAR-1991"
But I got the exception at least twice in build server, and got once in my local machine. 
It's better to separate this issue. but this task is out of time.
May I can create a bug and merge this task even if this are problem in build server

**Pull request merged**